### DISCUSSION TITLE
Life Itself lutruwita

### DISCUSSION THREAD

**Discussion by lianaguegan** [life-itself/community#1011] [General]
Hello dear Life Itself Community!

We are a tiny embryo group from lutruwita (palawa name for Tasmania in Australia - the palawa people are our local Indigenous people. Very wonderfully, many settler descendants are also now choosing to move away from colonial naming and honour first people namings). 

Our weeny group started out as a socio-cultural transformation discussion group as part of a Sociocracy Community Connections circle at The Reseed Centre, a centre for sustainability in north west lutruwita. As much of the content for our discussions came from Life Itself (LI) podcasts, and we were inspired by LI collective learning and praxis, it seemed fitting to consider kick starting a little pocket of LI here. So with LI’s blessings we are now at our very beginnings:

![IMG_0012](https://github.com/life-itself/community/assets/157182635/17516db1-b0b0-4ac2-b145-3bfd84595e4e)

We are starting our year with a LI Community Call recording: How to Handle Anything in Life and Community. Seems like a great way to get going!!!

While we will stick to sourcing a lot of our content from LI for contemplation and practice (hoping to run some small pop up residencies using LI course material), we will also engage with offerings from The Reseed Centre and call on various speakers and collective learning opportunities to nourish our culture creating.

Wish us luck!!

Warmest, liana

**Comment by rufuspollock** [comment]
@lianaguegan this is fantastic.

Let us know if you would like more suggestions for what you cover.

I also think you could try out using the Second Renaissance Introduction course (pilot version) as material for dialogs and discussions https://lifeitself.org/second-renaissance

**Comment by lianaguegan** [comment]
Thank you Rufus 🙏🏼🙏🏼I’ve just requested a review of the course. I would love to take this into our community, and yes please! any suggestions for what we cover, or could cover, would be very welcome.Warmest, liana 🙏🏼Sent from my iPhoneOn 23 Jan 2024, at 4:14 am,

**Reply by rufuspollock** [reply]
Ok, i'm cc'ing @laurenwigmore 

we'll arrange to share the links to the playlist of the course and we can maybe sync with Liana about setting her up to facilitate a group herself 😄
